storages/backends/gcloud.py

```diff
@@ -129,9 +133,17 @@ def get_default_settings(self):
             # roll over.
             "max_memory_size": setting('GS_MAX_MEMORY_SIZE', 0),
             "blob_chunk_size": setting('GS_BLOB_CHUNK_SIZE'),
-            "timeout": setting('GS_TIMEOUT', 60)
+            "timeout": setting('GS_TIMEOUT', 60),
+            "all_files_immutable": setting('GS_ALL_FILES_IMMUTABLE', False),
         }
 
+    @property
+    def retry_if_generation_specified_or_immutable(self):
```

Comment:
I find this name a bit confusing, I would call it get_retry_policy (like here: https://googleapis.dev/python/storage/latest/retry_timeout.html#google.cloud.storage.retry.DEFAULT_RETRY)